.gitignore

```diff
@@ -5,3 +5,4 @@
 /vendor-bin/**/vendor
 /vendor-bin/**/composer.lock
 /my127ws.phar
+/tests/Workspace
```

Comment:
Instead of using the system tmp dir, use a local dir - this makes inspecting the artifacts easier.